web/src/components/contactForm.js

```diff
@@ -0,0 +1,132 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import tw from 'twin.macro';
+
+const ContactWrapper = styled.div`
+    ${tw`text-center`}
+`;
+
+const StyledForm = styled.form`
+    ${tw`flex flex-col items-center`}
+`;
+
+const StyledHeading = styled.h1`
+    ${tw`font-bold mb-8 relative text-4xl text-black-3 leading-tight`}
+`;
+const InputWrapper = styled.div`
+    ${tw`flex gap-1`}
+`;
+
+const StyledButton = styled.button`
+    ${tw`cursor-pointer transition duration-500 ease-out bg-green py-2 px-6 text-white max-w-max mt-16 text-xl tracking-widest hover:bg-hoverGreen`}
+`;
+
+const StyledInput = styled.input`
+    ${tw`border-2 h-10 mt-2 py-1 px-2 text-lg text-black`}
+`;
+
+const StyledLabel = styled.label`
+    ${tw`flex flex-col mb-4 mx-4 text-sm text-left`}
+`;
+
+export default function ContactForm() {
+    const [formData, setFormData] = useState(null);
+    const [isSubmitFormSuccessful, setIsSubmitFormSuccessful] = useState(false);
+
+    const encode = (data) => {
+        return Object.keys(data)
+            .map(
+                (key) =>
+                    `${encodeURIComponent(key)}=${encodeURIComponent(
+                        data[key],
+                    )}`,
+            )
+            .join('&');
+    };
+
+    const handleChange = (event) => {
+        setFormData({ ...formData, [event.target.name]: event.target.value });
+    };
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        const form = e.target;
+        fetch('/', {
+            method: 'POST',
+            headers: { 'Content-Type': 'application/x-www-form-urlencoded' },
+            body: encode({
+                'form-name': form.getAttribute('name'),
+                ...formData,
+            }),
+        })
+            .then(() => setIsSubmitFormSuccessful(true))
+            .catch((error) => error);
+    };
+
+    return (
+        <ContactWrapper>
+            <StyledHeading>Request a Demo</StyledHeading>
+            {isSubmitFormSuccessful ? (
+                <StyledHeading>Thank you!</StyledHeading>
+            ) : (
+                <StyledForm
+                    method="post"
+                    onSubmit={handleSubmit}
+                    data-netlify="true"
+                    netlify-honeypot="bot-field"
+                >
+                    <p className="hidden">
```

Comment:
I don't think `hidden` is a tailwind class. Move this to a style component and apply the correct class.